mozdownload/scraper.py

```diff
@@ -46,7 +46,7 @@
                       'mac': r'mac',
                       'mac64': r'mac(64)?',
                       'win32': r'win32',
-                      'win64': r'win64-x86_64'}
+                      'win64': r'win64(-x86_64)?'}
```

Comment:
This was actually part of a PR I have landed yesterday. So this should cause a merge conflict.